acceptance-tests/mssql_test.go

```diff
@@ -14,4 +15,24 @@ var _ = Describe("MSSQL", Label("mssql"), func() {
 			services.WithPlan("default"))
 		defer serviceInstance.Delete()
 	})
+
+	FIt("can't be destroyed if `deletion_protection: true`", func() {
```

Comment:
nit-pick: remove the focus

Kind reminder:
```
ginkgo --label-filter=mssql --focus="can't be destroyed if `deletion_protection: true" -v acceptance-tests
```

Reply:
Fixed